deploy/template.yaml

```
@@ -209,33 +209,39 @@ Resources:
               SCENARIO: all
               K6_DYNATRACE_DASHBOARD_ID: 1e845440-d013-4472-9d65-2ea21a5cb41a
             parameter-store:
-              TEST_CLIENT_EXECUTE_URL: "/perfTest/testClientUrl"
-              BACK_END_URL: "/perfTest/backendUrl"
-              FRONT_END_URL: "/perfTest/frontendUrl"
-              BIOMETRIC_SESSION_ID: "/perfTest/biometricSessionID"
-              APP_PASSWORD: "/perfTest/testUserPassword"
-              AUTH_APP_KEY: "/perfTest/authAppKey"
-              APP_PASSWORD_NEW: "/perfTest/testUserNewPassword"
-              CORE_STUB_URL: "/perfTest/coreStubUrl"
-              KBV_URL: "/perfTest/kbvUrl"
-              ADDRESS_URL: "/perfTest/addressUrl"
-              ADDRESS_ENV_NAME: "/perfTest/addressEnvName"
-              KBV_ENV_NAME: "/perfTest/kbvEnvName"
-              CORE_STUB_USERNAME: "/perfTest/coreStubUsername"
-              CORE_STUB_PASSWORD: "/perfTest/coreStubPassword"
-              CORE_URL: "/perfTest/coreUrl"
-              ORCH_STUB_URL: "/perfTest/orchStubUrl"
-              RP_STUB: "/perfTest/rpStub"
-              BASE_URL: "/perfTest/baseUrl"
-              F2F_URL: "/perfTest/f2fUrl"
-              K6_DYNATRACE_URL: "/perfTest/dynatraceUrl"
-              HOME_URL: "/perfTest/homeUrl"
-              SIGNIN_URL: "/perfTest/signInUrl"
-              EMAIL_OTP: "/perfTest/fixedEmailOTP"
-              PHONE_OTP: "/perfTest/fixedPhoneOTP"
-              CURR_PHONE: "/perfTest/currPhone"
-              NEW_PHONE: "/perfTest/newPhone"
+              ACCOUNTS_APP_PASSWORD: "/perfTest/account/testUserPassword"
+              ACCOUNTS_APP_PASSWORD_NEW: "/perfTest/account/testUserNewPassword"
+              ACCOUNTS_APP_KEY: "/perfTest/account/accounts/appKey"
+              ACCOUNTS_AUTH_APP_KEY: "/perfTest/account/accounts/authAppKey"
```

```suggestion
              ACCOUNTS_AUTH_APP_KEY: "/perfTest/account/accounts/authAppKey"
```
The first one here is a duplicate of the second, right?

---

Yes they are the same, do you want to combine them and let only one?  ACCOUNT_APP_KEY: "/perfTest/account/account_app_key"?

---

Yes, let's remove duplication where we can